doc/Tutorials/index.rst

```diff
@@ -9,7 +9,7 @@ you may also try out live copies on `mybinder.org
 explore many other examples in our `contrib repository
 <https://github.com/ioam/holoviews-contrib>`_. Lastly, for the most
 responsive experience, you can install HoloViews and try out the
-`tutorial notebooks`_ for yourself.
+tutorial notebooks for yourself.
 
```

Comment:
It works just fine which is why the dev version is correctly pointing to 1.6.2 - the issue is that we need to copy the files from the dev version over to the main website on release. I *think* what happened is that the website building infrastructure broke so that the files must have been stale.

What I do know is that there are tricky bits to updating the docs/website but thankfully this isn't one of them! :-)